### PR TITLE
Set up a "Hello world" project with two apps 🥇 

### PR DESCRIPTION
**Pull request by ismailmunyentwari9**
# What I have done in this project 🥇 

- [ ] Create a new Rails API app called 'hello-rails-back-end'.
- [ ] Initialize  project with Git.
- [ ] Make sure that  project has a Postgres database set up.
- [ ] Create a table for storing  messages and create 5 different greetings.
- [ ] Create an API endpoint that selects a random greeting
# 🧠  skills gained 🖲️ 

- [ ] Implement a connection between a Ruby on Rails back-end and React front-end.
- [ ] Understand pros and cons of different approaches of connecting Ruby on Rails back-end with React front-end.

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @ismailmunyentwari9,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### You should have an open pull request in the frontend repo containing the changes. Kindly add that.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by DammyShittu — COMMENTED** [review]
Hi @ismailmunyentwari9,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Your frontend repo does not have a descriptive readme documentation

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,116 @@
...
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/REACT">REACT</a></li>
+  </ul>
+</details>
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/REDUX">REDUX</a></li>
+  </ul>
+</details>
```
Comment (abdulhamiid) [inline README.md]: I notice that all the links in the `Tech Stack` is broken, kindly fix this please, let's avoid broken links in our README for a professional repository 🤗

File: app/controllers/api/greetings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Api::GreetingsController < ApplicationController
+  def random
+    greeting = Salute.order('RANDOM()').first
+    render json: { message: greeting.name }
+  end
+end
```
Comment (abdulhamiid) [inline app/controllers/api/greetings_controller.rb]: image

Your API endpoint selects a random greeting from your table 💯 👍

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @ismailmunyentwari9 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Use GitHub Flow
- ✔️ Create an API endpoint

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by afanimartin — APPROVED** [review]
Hi @ismailmunyentwari9, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Broken links fixed

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**